h2/src/docsrc/help/help.csv

```diff
@@ -1862,6 +1862,14 @@ This command commits an open transaction in this session.
 SET SESSION CHARACTERISTICS AS TRANSACTION ISOLATION LEVEL SERIALIZABLE
 "
 
+"Commands (Other)","SET STATEMENT_TIMEOUT","
+@h2@ SET STATEMENT_TIMEOUT int
+","
+Synonym for SET QUERY_TIMEOUT, for PostgreSQL compatibility.
+","
+SET STATEMENT_TIMEOUT 10000
+"
+
```

Comment:
Please, remove it from the documentation. We don't want to support this non-standard command.

Reply:
removed